students/christopher_gantt/lesson05/mailroom.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import sys
+
+donor_db = {"Wassily Kandinsky": [43928.13, 131.34, 1928.0],
+            "Jasper Johns": [3134.43, 153.34],
+            "Mark Rothko": [135353.33],
+            "Richard Serra": [153757.87, 28457.12, 1293451.0],
+            "Yves Tanguy": [1534.23, 2542.19]}
+
+PROMPT = "\n".join(("\nMain Menu", 
+        "Type a number from the options below",
+        "1 - Send a Thank You to a single donor",
+        "2 - Create a Report",
+        "3 - Send letters to all donors",
+        "4 - Quit\n"))
+
+
+def thank_you(): 
+    '''
+    I could use a try block with this function, doing something very similar to my main function, creating a sub-prompt
+    and multiple functions. I've decided against this here though because it elongates the choosing process for the user
+    and doesn't seem to actually protect against user typing errors any better than this code already does. If i wanted
+    to address that matter I might add a remove entry function. 
+    In summary, as per the exercise prompt, "Improve your mailroom by adding exceptions and comprehensions.", I do not 
+    believe that changing this code structure will improve my mailroom.
+    '''
+    thanks = True
+    while thanks:
+        user_in = input("Type List to view all donor names, type a current donor name, or type the name of a new donor: ").title()
+        if user_in == "List": 
+            [print(key) for key in donor_db]
```

Comment:
this is considered an anti-pattern

comprehension will allocate a list of None values (because `print()` returns None)
instead you want this: `print("\n".join([<comprehension_here>]))`